com.abagames.noiz2sa.yaml

```diff
@@ -0,0 +1,89 @@
+app-id: com.abagames.noiz2sa
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: run-noiz2sa.sh
+rename-desktop-file: noiz2sa.desktop
+rename-icon: noiz2sa
+rename-appdata-file: noiz2sa.appdata.xml
+finish-args:
+- "--device=dri"
+- "--share=ipc"
+- "--socket=pulseaudio"
+- "--socket=x11"
+- "--persist=.noiz2sa.prf"
+modules:
+- shared-modules/glu/glu-9.json
+- shared-modules/SDL/SDL-1.2.15.json
+- shared-modules/SDL/SDL_mixer-1.2.12.json
+- name: bulletml
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: http://http.debian.net/debian/pool/main/b/bulletml/bulletml_0.0.6.orig.tar.gz
+    sha256: 162544e708cd8f61c944f7eac2a0403d2900d29fa9b1dc45235cfccfac8ae960
+  - type: patch
+    path: patches/bulletml/fixes.patch
+  - type: patch
+    path: patches/bulletml/bulletml_d.patch
+  - type: patch
+    path: patches/bulletml/d_cpp.patch
+  - type: patch
+    path: patches/bulletml/warnings.patch
+  - type: patch
+    path: patches/bulletml/makefile.patch
+  - type: patch
+    path: patches/bulletml/includes.patch
+  - type: patch
+    path: patches/bulletml/get-rid-of-boost.patch
+  build-commands:
+  - make -C src CXXFLAGS="$(CXXFLAGS) $(CPPFLAGS) -fPIC -fpic" LDFLAGS="$(LDFLAGS)
+    -Wl,-z,defs" libbulletml.so
+  - mkdir -p /app/include/d/ /app/include/bulletml/ /app/include/bulletml/tinyxml/
+    /app/lib
+  - cp src/bulletml.d /app/include/d/
+  - cp src/*.h /app/include/bulletml/
+  - cp src/tinyxml/*.h /app/include/bulletml/tinyxml/
+  - cp src/lib*.so* /app/lib/
```

Comment:
there should be a `cleanup` rule to remove the content of `/include` after the build.

Reply:
@joaquimrocha Move the cleanup to a global one. SDL leaves some leftovers in `/include`. Also add `/lib/pkgconfig`, `*.pc`

I'll fix this in the shared-modules later.

Reply:
> I'll fix this in the shared-modules later.

Merged now, you just need to update the shared-modules submodule. Keep rest as is.